.devcontainer/devcontainer.json

```diff
@@ -22,8 +22,8 @@
 	"forwardPorts": [3000],
 
 	// Specifies a command that should be run after the container has been created.
-	"postCreateCommand": "npm install",
+	"postCreateCommand": "npm install && npm run prepare",
```

Comment:
[F] For v6 of husky we have to run an extra script. Husky dropped support for autoinstall when they moved to v6. Reasoning is detailed [here](https://blog.typicode.com/husky-git-hooks-autoinstall/).

Reply:
Gotcha that makes sense!